ci/matrix.yaml

```
@@ -11,6 +11,49 @@ workflows:
 
   pull_request:
     # Old CTK
+    - {jobs: ['build'], std: 'all', ctk: '11.1', cxx: ['gcc6', 'clang9', 'msvc2017']}
+    - {jobs: ['build'], std: 'all', ctk: '11.8', cxx: ['gcc11'], sm: '60;70;80;90'}
+    # Current CTK build-only
+    - {jobs: ['build'], std: 'all', cxx: ['gcc7', 'clang9', 'msvc2019']}
+    - {jobs: ['build'], std: 'all', cxx: ['gcc', 'clang', 'msvc', 'intel']}
+    # Current CTK testing:
+    - {jobs: ['test'],  project: ['libcudacxx', 'thrust'], std: 'minmax', cxx: ['gcc']}
+    - {jobs: ['test'],  project: ['libcudacxx', 'thrust'], std: 'max',    cxx: ['clang', 'msvc']}
+    # Split up cub tests:
+    - {jobs: ['test_nolid', 'test_lid0'], project: ['cub'], std: 'minmax', cxx: ['gcc']}
+    - {jobs: ['test_lid1',  'test_lid2'], project: ['cub'], std: 'max',    cxx: ['gcc']}
+    - {jobs: ['test_nolid', 'test_lid0'], project: ['cub'], std: 'max',    cxx: ['clang', 'msvc']}
+
+    # Modded builds:
+    - {jobs: ['build'], std: 'max', cxx: ['gcc', 'clang'], cpu: 'arm64'}
+    - {jobs: ['build'], std: 'max', cxx: ['gcc'], sm: '90a'}
+    # default_projects: clang-cuda
+    - {jobs: ['build'], std: 'max', cudacxx: 'clang', cxx: 'clang'}
+    # nvrtc:
+    - {jobs: ['nvrtc'], project: 'libcudacxx', std: 'minmax'}
```

There is only one nvrtc job so I would really like full coverage here
```suggestion
    - {jobs: ['nvrtc'], project: 'libcudacxx', std: 'all'}
```

---

This will cost a few more GPU jobs, but seems reasonable.